ios/RNCPagerViewComponentView.mm

```diff
@@ -76,6 +80,7 @@ - (instancetype)initWithFrame:(CGRect)frame
         _destinationIndex = -1;
         _layoutDirection = @"ltr";
         _overdrag = NO;
+        _scrollEnabled = YES;
```

Comment:
When adopting this code to patch version 6.9.1 in our app I had to remove this line to make it work. Don't know if you need to do the same in this PR but just wanted to share my findings and give you a heads up. Thank you for fixing this!

Reply:
Thanks for checking it out! I'm not sure why it wouldn't work for you.. Without setting this to YES the state will be desynchronized (as the default for a scroll view is to have scroll enabled). 

I did some additional tests and it was working well with this.